backend/test/test_helper.clj

```diff
@@ -1,6 +1,37 @@
 (ns test-helper
-  (:require [configs.hands :as hands]))
+  (:require [configs.hands :as hands]
+            [api.base :as api]
+            [rules.create-game :as create-game]
+            [rules.play-card :as play-card]))
 
 (defn default-hand-power
   [x]
   (get-in hands/default-hand [x :power]))
+
+(defn play-as-api
```

Comment:
I feel these two functions can be abstracted together, maybe by using a third function to build on.